Microsoft.Toolkit.Uwp.UI.Controls.Core/DropShadowPanel/DropShadowPanel.cs

```diff
@@ -168,7 +167,11 @@ private void UpdateShadowMask()
             {
                 CompositionBrush mask = null;
 
-                if (Content is Image)
+                if (Content is IAlphaMaskProvider maskedControl)
```

Comment:
We probably want this if condition last as it's probably going to be the least common?

Reply:
You are right in that it will be the last common, but that would mean we wouldn't use the custom masks when Content happens to be derived from one of the other classes, e.g Content could be an instance of a class that derives from Windows.UI.Xaml.Shapes.Shape, but can't still can't override GetAlphaMask because its not marked as virtual.

Reply:
Ah, right good point! Thanks for the explanation. Feel free to add a code comment to just call out we that in brief like 'if someone has implemented this we want to prioritize'. Just in case in the future we forget 😉